src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/PluginConfig.java

```diff
@@ -178,4 +201,95 @@ public int getReplicationCacheExpirationInMinutes() {
     public void setReplicationCacheExpirationInMinutes(int replicationCacheExpirationInMinutes) {
         this.replicationCacheExpirationInMinutes = replicationCacheExpirationInMinutes;
     }
+
+    @Override
+    public int getEventTypesSize() {
+        return GerritEventType.values().length;
+    }
+
+    @Override
+    public List<String> getFilterIn() {
+        if (filterIn == null) {
+            GerritEventType[] types = GerritEventType.values();
+            List<String> typeList = new ArrayList<>();
+            for (GerritEventType type : types) {
+                if (type.isInteresting()) {
+                    typeList.add(type.getTypeValue());
+                }
+            }
+            Collections.sort(typeList);
+            return typeList;
+        }
+        return filterIn;
+    }
+
+    /**
+     * Get the filter event list from formdata.
+     * @param formData the JSON object with form data.
+     * @return value.
+     */
+    private List<String> getFilterInFromFormData(JSONObject formData) {
+        // Getting the list as a JSONArray is not utilized since if the string is empty
+        // the get method will throw an exception. The logic is such that if the filter
+        // is null then the default of every event being interesting is assumed.
+        String stringIn = formData.optString("filterIn");
+        List<String> filter = null;
+        String[] arrayIn = null;
+        if (!stringIn.equals("null")) {
+            filter = new ArrayList<>();
+            arrayIn = stringIn.split(" ");
```

Comment:
might be better to split on `\\s+` to cover any white space.

Reply:
Done